friend/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+import Friends from "./Friends/friends";
+
+class App extends Component {
+  constructor (){
+    super();
+    this.state = {
+      friends: [],
+      name: '',
+      age: '',
+      email: ''
```

Comment:
Properties in state have intuitive names and are initialized with the right types, very good!